tests/selenium/test_sizing.py

```diff
@@ -0,0 +1,270 @@
+                for i in variations_range
+            ]
+        elif n_clicks == 3:
+            return [
+                dict(
+                    width=750,
+                    minWidth=750,
+                    maxWidth=750,
+                    display="none",
+                    paddingBottom=10,
+                )
+                for i in variations_range
+            ]
+        elif n_clicks:
+            return [
+                dict(width=350, minWidth=350, maxWidth=350, paddingBottom=10)
+                for i in variations_range
+            ]
+
+    test.start_server(app)
+
+    targets = [None, "table500", "table750", None, "table350"]
+
+    for targetId in targets:
+        target = (
+            test.driver.find_element_by_css_selector("#{}".format(targetId))
+            if targetId is not None
+            else None
+        )
+
+        for variation in variations:
+            table = test.driver.find_element_by_css_selector(
+                "#{}".format(variation["id"])
+            )
+            if targetId is None:
+                assert table is not None
+                assert (
+                    test.driver.execute_script(
+                        "return getComputedStyle(document.querySelector('#{} .dash-spreadsheet-container')).display".format(
+                            variation["id"]
+                        )
+                    )
+                    == "none"
+                )
+            else:
+                cells_are_same_width(target, table)
+
+        test.driver.find_element_by_css_selector("#btn").click()
+
+
+def test_szng002_percentages_result_in_same_widths(test):
+    base_props = dict(
+        columns=[
+            {"id": "_", "name": ["_", "_", "_"]},
+            {
+                "id": "a",
+                "name": [
+                    "A-----------------VERY LONG",
+                    "A-----------------VERY LONG",
+                    "A-----------------VERY LONG",
+                ],
+            },
+            {
+                "id": "b",
+                "name": [
+                    "A-----------------VERY LONG",
+                    "A-----------------VERY LONG",
+                    "B-----------------VERY LONG",
+                ],
+            },
+            {
+                "id": "c",
+                "name": [
+                    "A-----------------VERY LONG",
+                    "B-----------------VERY LONG",
+                    "C-----------------VERY LONG",
+                ],
+            },
+        ],
+        data=[
+            {"_": 0, "a": 85, "b": 601, "c": 891},
+            {"_": 0, "a": 967, "b": 189, "c": 514},
+            {"_": 0, "a": 398, "b": 262, "c": 743},
+            {"_": 0, "a": 89, "b": 560, "c": 582},
+            {"_": 0, "a": 809, "b": 591, "c": 511},
+        ],
+        style_table=dict(width=500, minWidth=500, maxWidth=500, paddingBottom=10),
+        style_cell=dict(width="20%", minWidth="20%", maxWidth="20%"),
```

Comment:
I only see 4 columns - is it intentional to have <100% total width?

Reply:
https://github.com/plotly/dash-table/pull/793/commits/575aa2c4bc838474bcfa251c8596897be6c76d17#diff-cd5fbde6c058807ca1b16f17f7daa4d8R213